adi/adis16475.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2019-2023 Analog Devices, Inc.
+#
+# SPDX short identifier: ADIBSD
+
+from adi.context_manager import context_manager
+from adi.rx_tx import rx
+
+
+class adis16475(rx, context_manager):
+    """ADIS16475 Compact, Precision, Six Degrees of Freedom Inertial Sensor"""
+
+    _complex_data = False
+    _rx_channel_names = [
+        "anglvel_x",
+        "anglvel_y",
+        "anglvel_z",
+        "accel_x",
+        "accel_y",
+        "accel_z",
+        "temp0",
+    ]
+    _device_name = ""
+
+    def __init__(self, uri="", device_name="adis16505-2"):
+        context_manager.__init__(self, uri, self._device_name)
+
+        compatible_parts = [
+            "adis16470",
```

Comment:
Please add this list to the supported parts md file.

Reply:
Should I do anything else besides adding ADIS16475 to the list?